docs/operational_guide/namespace_changes.md

```diff
@@ -82,7 +82,7 @@ TODO
 
 The operations below include sample CURLs, but you can always review the API documentation by navigating to
 
-`http://<M3_COORDINATOR_IP_ADDRESS>:<CONFIGURED_PORT(default 7201)>/api/v1/openapi`
+`http://<M3_COORDINATOR_HOST_NAME>:<CONFIGURED_PORT(default 7201)>/api/v1/openapi` or our [online API documentation](https://m3db.io/openapi/).
```

Comment:
unrelated to your change but the deleting a namespace HTTP endpoint isn't correctly quoted: https://github.com/m3db/m3/blob/2eba3b8e0abc28b962d9d0132a7b0e82328a5c4e/docs/operational_guide/namespace_changes.md. mind updating.